### PR TITLE
Update bulk-export-data.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
DM/GM Channels and Their Posts in the Bulk Export: https://github.com/mattermost/mattermost-server/pull/10421

### CODE REVIEW

File: source/administration/bulk-export-data.rst [hunk 1]
```diff
@@ -14,6 +14,8 @@ You can export the following data types:
 - Posts' Replies
 - Posts' Reactions
 - Custom Emoji
+- Direct Channels
```
Comment (jasonblais) [inline source/administration/bulk-export-data.rst]:
```suggestion
- Direct Message Channels 
```

File: source/administration/bulk-export-data.rst [hunk 2]
```diff
@@ -14,6 +14,8 @@ You can export the following data types:
 - Posts' Replies
 - Posts' Reactions
 - Custom Emoji
+- Direct Channels 
+- Direct Posts
```
Comment (jasonblais) [inline source/administration/bulk-export-data.rst]:
```suggestion
- Direct Message Posts
```

File: source/administration/bulk-export-data.rst [hunk 3]
```diff
@@ -565,3 +567,51 @@ Emoji object
...
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">members</td>
+      <td valign="middle">array</td>
+      <td>A List of members.</td>
```
Comment (jasonblais) [inline source/administration/bulk-export-data.rst]:
```suggestion
      <td>List of channel members.</td>
```

**Comment by wiersgallak** [comment]
Thanks @jasonblais  - I addressed your edits.

**Pull request merged**